client/command/info/info.go

```diff
@@ -120,9 +124,42 @@ func WhoamiCmd(ctx *grumble.Context, con *console.SliverConsoleClient) {
 	if session == nil && beacon == nil {
 		return
 	}
+
+	var isWin bool
+	con.Printf("Logon ID: ")
 	if session != nil {
 		con.Printf("%s\n", session.Username)
+		if session.GetOS() == "windows" {
+			isWin = true
+		}
 	} else if beacon != nil {
 		con.Printf("%s\n", beacon.Username)
+		if beacon.GetOS() == "windows" {
+			isWin = true
+		}
+	}
+
+	if isWin {
+		con.Printf("Current Token ID: ")
```

Comment:
This will work fine in session mode, but not in beacon mode: this line will be printed, but no information about the token ID will be displayed until the beacon reaches out and the callback triggers.
I recommend moving this into some `PrintToken` dedicated function that takes the `cto.Output` and prints it. That way you can call it in both cases.